src/applications/personalization/dashboard/tests/components/health-care-v2/HealthCareV2.unit.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { expect } from 'chai';
+import { renderWithStoreAndRouter } from '~/platform/testing/unit/react-testing-library-helpers';
+
+import { UnconnectedHealthCareV2 } from '../../../components/health-care-v2/HealthCareV2';
+
+describe('<UnconnectedHealthCareV2 />', () => {
+  it('should render', () => {
+    const tree = renderWithStoreAndRouter(<UnconnectedHealthCareV2 />, {});
+
+    tree.getByText('Health care');
+    tree.getByTestId('dashboard-section-health-care-v2');
+  });
+
+  it('should have the correct header class names when loading', () => {
+    const tree = renderWithStoreAndRouter(
+      <UnconnectedHealthCareV2 shouldShowLoadingIndicator />,
+      {},
+    );
+
+    tree.getByText('Health care');
+    tree.getByTestId('dashboard-section-health-care-v2');
+    expect(
+      tree.container.querySelector(
+        'h2.vads-u-margin-top--0.vads-u-margin-bottom--2',
+      ),
+    ).to.exist;
```

Comment:
I'm not sure if this would be something we would want to test in this way.

If we decide to style the heading slightly differently using an additional utility class for whatever reason, then this test would start failing, and that could become a burden. Maybe some other logic should be tested? If you conditionally set the `data-testid` could you test the loading logic without being reliant on the class names specifically?